src/components/StatusBarsForNormalization.jsx

```diff
@@ -1,41 +1,53 @@
-import React from "react";
+import React, {useState, useEffect} from 'react';
 import UnAssigned from "../logos/UnAssigned.svg";
 import Assigned from "../logos/Assigned.svg";
 import InProgress from "../logos/InProgress.svg";
 import Completed from "../logos/Completed.svg";
 import StatusBar from "./StatusBar";
+import {workFlowsCountUrl} from '../constants/index'
 
 function StatusBarsForNormalization() {
+  const [statusBarData, setStatusBarData] = useState([]);
+
+ useEffect(() => {
+        fetch(workFlowsCountUrl,{
```

Comment:
While you are fetching this, are you showing any loaders in the statusbar component ?

Reply:
no we don't